docs/reference/sdk/functions/assertCondition.md

```diff
@@ -1,18 +1 @@
-# Function: assertCondition
-
-▸ **assertCondition**(`condition`, `message?`): asserts condition
-
-#### Parameters
-
-| Name | Type |
-| :------ | :------ |
-| `condition` | `any` |
-| `message?` | `string` |
-
-#### Returns
-
-asserts condition
-
-#### Defined in
-
-[helpers/ensure.ts:25](https://github.com/coda/packs-sdk/blob/main/helpers/ensure.ts#L25)
+REFLECTION_MEMBER
```

Comment:
It looks like perhaps upgrading the typescript markdown library broke some of the docs?